packages/cli-platform-ios/src/tools/checkIfConfigurationExists.ts

```diff
@@ -1,10 +1,15 @@
-import {CLIError} from '@react-native-community/cli-tools';
+import {CLIError, logger} from '@react-native-community/cli-tools';
 import {IosProjectInfo} from '../types';
 
 export function checkIfConfigurationExists(
   project: IosProjectInfo,
   mode: string,
 ) {
+  if (!project) {
+    logger.warn(`Unable to check whether "${mode}" exists in your project`);
```

Comment:
I don't know if this warning is actually useful. I'd prefer to not have it here, but I'll let you decide.